ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeOwnersReview.vue

```diff
@@ -6,22 +6,25 @@
     </header>
 
     <div :class="{ 'border-error-left': !getStepValidation(MhrSectVal.HOME_OWNERS_VALID) }">
-      <div v-show="!getStepValidation(MhrSectVal.HOME_OWNERS_VALID)" class="px-6 py-8">
-        <v-icon color="error">mdi-information-outline</v-icon>
-        <span class="error-text mx-1">This step is unfinished.</span>
-        <router-link :to="{ path: `/${RouteNames.MHR_REGISTRATION}/${RouteNames.HOME_OWNERS}` }"
-          >Return to this step to complete it.
-        </router-link>
-      </div>
-      <section class="px-6 my-2" v-if="hasHomeOwners || (hasGroups && showGroups)">
+      <section v-show="!getStepValidation(MhrSectVal.HOME_OWNERS_VALID)"
+      :class="hasHomeOwners ? 'pt-30px px-6' : 'px-6 py-8'">
+        <span>
+          <v-icon color="error">mdi-information-outline</v-icon>
+          <span class="error-text mx-1">This step is unfinished.</span>
+          <router-link :to="{ path: `/${RouteNames.MHR_REGISTRATION}/${RouteNames.HOME_OWNERS}` }"
+          >Return to this step to complete it.</router-link>
+        </span>
+      </section>
+
+      <section class="px-6 my-2" v-if="hasHomeOwners">
```

Comment:
What displays here if this is `false` 
I guess previously there was fallback values below for missing information.

Reply:
If this is false only the error message will display, as per requirements in the ticket.
![image](https://user-images.githubusercontent.com/112968185/199819155-013af67d-f575-40b5-bc0f-8e0de5871819.png)

Reply:
ok fair enough, requirements maybe updated?

That being said, are the fallback values still utilized/required below?

Reply:
Not sure on what you mean by fallback values...

Reply:
Basically a fallback value is an `||` scenario. Ie render `someProperty` || 'N/A'

Reply:
If you mean the template within this v-if, then yes it is used to display the table on R/C when owners are present

Reply:
ohhhh gotcha, then I guess there's no case in which tenancy type can be N/A but still have an owner present. I'll remove the fallbacks